002_simulation_structure/002_simulation_structure.tex

```diff
@@ -1,23 +1,109 @@
 \chapter{Simulation Structure}\label{simulation_structure}
-
 %%%%%%%%
-%%%%%%%% Simulation Environment
+%%%%%%%% Scenario and Simulator
 %%%%%%%%
-\section{Simulation Environment}\label{simulation_environment}
-Goes over the backend design
+\section{Scenario and Simulator}\label{scenario_and_simulator}
 
-%%%%%%%%
-%%%%%%%% Simulation Flow
-%%%%%%%%
-\section{Simulation Flow}\label{simulation_flow}
-The order in which events happen
-Need a nice diagram for this
+\subsection{Context}
+
+The Platform presents an environment consisting of people subjected to a life within a tower. This tower has a predefined set of laws that closely control the people’s lives, and it is ultimately up to these individuals to act however they suppose is correct.
```

Comment:
'This tower has a predefined set of laws that closely control the people’s lives,' is this true??

Reply:
No? I mean its a very restricted environment, maybe the wording was a bit extreme but I think its alright....